xtask/src/install.rs

```diff
@@ -180,7 +180,7 @@ fn install_server(opts: ServerOpt) -> Result<()> {
         Malloc::Jemalloc => &["--features", "jemalloc"],
     };
 
-    let cmd = cmd!("cargo install --path crates/rust-analyzer --locked --force {features...}");
+    let cmd = cmd!("cargo install --path crates/rust-analyzer --locked --force --features force-always-assert {features...}");
```

Comment:
Ugh, I spent quite a bit of time thinking I broke feature autocomplete in #7797 when it was actually panicking.